src/ray/core_worker/common.h

```diff
@@ -97,6 +95,48 @@ enum class StoreProviderType { PLASMA };
 
 enum class TaskTransportType { RAYLET };
 
+/// Translate from ray::rpc::Language to Language type (required by raylet client).
+///
+/// \param[in] language Language for a task.
+/// \return Translated task language.
+inline ::Language ToTaskLanguage(ray::rpc::Language language) {
+  switch (language) {
+  case ray::rpc::Language::JAVA:
+    return ::Language::JAVA;
+    break;
+  case ray::rpc::Language::PYTHON:
+    return ::Language::PYTHON;
+    break;
+  case ray::rpc::Language::CPP:
+    return ::Language::CPP;
+    break;
+  default:
+    RAY_LOG(FATAL) << "invalid language specified: " << static_cast<int>(language);
```

Comment:
invalid -> Invalid

We should log the meaningful name instead of an int?

Reply:
You can't get a meaningful name if the value is out of valid range of enum. e.g. `static_cast<ray::rpc::Language>(100)`. And eventually, the old, flatbuffer generated definition of language will be replaced by protobuf generated one. So these two conversion functions will go away in the near future.